### PR TITLE
Update Jquery fix for Letsbeclear

### PR DESCRIPTION
**Pull request by vinhtvu2**
Remove older version of JQuery to enqueue newer version.

Copied code fix from this repo:
https://github.com/LaughingSamurai/UCFDiversity/commit/e513d292fe5e81cb7380339fab58925fd441e76f

### CODE REVIEW

File: templates/head.php [hunk 1]
```diff
@@ -28,8 +28,12 @@
   <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
   <script>window.jQuery || document.write('<script src="<?php echo get_template_directory_uri(); ?>/assets/js/vendor/jquery-1.11.2.min.js"><\/script>')</script>
```
Comment (cjg89) [inline templates/head.php]: jQuery 1.11.2 inclusion should be removed here

File: lib/cleanup.php [hunk 2]
```diff
@@ -83,18 +83,6 @@ function roots_language_attributes() {
 
 add_filter('language_attributes', 'roots_language_attributes');
 
-/**
- * Clean up output of stylesheet <link> tags
- */
-function roots_clean_style_tag($input) {
```
Comment (cjg89) [inline lib/cleanup.php:89]: Is this code deletion necessary if the only intended change is a jQuery bump?

File: lib/init.php [hunk 3]
```diff
@@ -22,7 +22,7 @@ function roots_setup() {
   // add_theme_support('post-formats', array('aside', 'gallery', 'link', 'image', 'quote', 'status', 'video', 'audio', 'chat'));
 
   // Tell the TinyMCE editor to use a custom stylesheet
-  add_editor_style('assets/css/editor-style.css');
+  // add_editor_style('assets/css/editor-style.css');
```
Comment (cjg89) [inline lib/init.php]: Is this code commenting necessary if the only intended change is a jQuery bump?

File: templates/head.php [hunk 4]
```diff
@@ -28,8 +28,12 @@
...
 
+  <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
```
Comment (cjg89) [inline templates/head.php:28]: Unless I'm missing something, it doesn't look like this site uses Athena--are the tether.js and Athena JS inclusions necessary?

**Review by cjg89 — CHANGES_REQUESTED** [review]

File: templates/head.php (hunk 1 above)
Comment (vinhtvu2) [reply templates/head.php]: Removed on new push.

File: lib/cleanup.php (hunk 2 above)
Comment (vinhtvu2) [reply lib/cleanup.php:89]: This code removal was necessary after updating beyond WordPress 5.3:
https://hirecollin.com/2019/11/how-to-fix-broken-sage-roots-themes-that-broke-in-wordpress-5-3-0/

File: lib/init.php (hunk 3 above)
Comment (vinhtvu2) [reply lib/init.php]: This is unnecessary removal. Added back.

**Review by rjucf — APPROVED** [review]

File: templates/head.php (hunk 4 above)
Comment (vinhtvu2) [reply templates/head.php:28]: Removed Tether and Athena.

**Comment by vinhtvu2** [comment]
Fixed some of the issues Jo brought up in the PR :-)
Thanks Jo!

**Review by cadie — APPROVED** [review]

**Review by jmbarne3 — APPROVED** [review]

**Review by cjg89 — APPROVED** [review]

**Pull request merged**